build.zig

```diff
@@ -156,6 +166,19 @@ pub fn build(b: *zbs.Builder) !void {
     }
 }
 
+fn writePkgconfigFile(writer: anytype, prefix: []const u8) !void {
+    try writer.print(
+        \\prefix={s}
+        \\datarootdir=${{prefix}}/share
+        \\pkgdatadir=${{datarootdir}}/river-protocols
+        \\
+        \\Name: River Protocols
+        \\URL: https://github.com/ifreund/river
+        \\Description: River protocol files
+        \\Version: 0.1.0
```

Comment:
why don't we use the `full_version` here?

Reply:
Oh right of course, I hadn't thought about it